weather.py

```diff
@@ -57,7 +60,21 @@ def load_data_from_csv(csv_file):
     Returns:
         A list of lists, where each sublist is a (non-empty) line in the csv file.
     """
-    pass
+    data = []
+    with open(csv_file, encoding="utf-8") as csvfile:
+        reader = csv.reader(csvfile)
+        for row in reader:
+            if row == []: #getting rid of empty list
+                pass
+            elif row == ["date", "min", "max"]: #getting rid of header
+                pass
```

Comment:
This works, an alternative is to use `next(reader)` to skip the header without needing to know what the exact values are. This might be more useful in cases where we have many header fields, or the field values are long/contain complex characters.